pages/Nvidia/_index.md

```diff
@@ -1,14 +1,12 @@
 # Foreword
 
-There is no _official_ support for Nvidia. Unfortunately, their drivers are so
-messy, and their products so random, that it's impossible for us to help if
-these instructions don't work fully.
+There is no _official_ Hyprland support for Nvidia hardware. However, you might make it work properly following this page. 
 
-Every card seems to be random, and might work perfectly, or not work at all.
+You can choose between the proprietary [Nvidia drivers](https://wiki.archlinux.org/title/NVIDIA) or the open source [Nouveau driver](https://wiki.archlinux.org/title/Nouveau). Under the proprietary Nvidia drivers category, there are 3 of them: the current driver named 'nvidia' (or 'nvidia-dkms' to use with custom linux kernels) which is on active development, the legacy drivers 'nvidia-3xxxx' for older cards which Nvidia no longer actively supports, and the 'nvidia-open' driver which is currently an alpha stage attempt to open source part of their close source driver for newer cards.
```

Comment:
`is on active development` -> `is under active development`
`to open source part of their` -> `to open source a part of their`